MAUI/Calendar/events.md

```diff
@@ -179,3 +179,236 @@ private void OnCalendarLongPressed(object sender, CalendarLongPressedEventArgs e
 {% endhighlight %}
 {% endtabs %}
 
+## Commands
+
+### ViewChangedCommand
+
+SfCalendar includes a built-in event called `ViewChanged` and invokes this event through the [ViewChangedCommand]() whenever the view is navigated to either the previous or next view. Additionally, the event is triggered when switching between different calendar views, with the CalendarViewChangedEventArgs provided as a parameter.
+
+{% tabs %}
+{% highlight xaml tabtitle="MainPage.xaml" hl_lines="2" %}
+
+<calendar:SfCalendar  x:Name="Calendar" 
+                      ViewChangedCommand="ViewChangedCommand">
+<ContentPage.BindingContext>
+    <local:CalendarViewModel/>
+</ContentPage.BindingContext>					  
+</calendar:SfCalendar>
+
+{% endhighlight %}
+{% highlight c# tabtitle="MainPage.xaml.cs" hl_lines="3,6,8" %}
+
+public class CalendarViewModel
+{
+    public ICommand ViewChangedCommand { get; set; }
+    public CalendarViewModel()
+    {
+        ViewChangedCommand = new Command<CalendarViewChangedEventArgs>(ViewChanged);
+    }
+    private void ViewChanged(CalendarViewChangedEventArgs args)
+    {
+        // To do your requirement here.
+    }
+}
+
+{% endhighlight %}
+{% endtabs %}
+
+### SelectionChangedCommand
+
+SfCalendar includes a built-in event called `SelectionChanged` and invokes this event through the [SelectionChangedCommand]() whenever selection changes in the calendar  changes, with the `CalendarSelectionChangedEventArgs` passed as a parameter.
+
+{% tabs %}
+{% highlight xaml tabtitle="MainPage.xaml" hl_lines="2" %}
+
+<calendar:SfCalendar  x:Name="Calendar" 
+                     SelectionChangedCommand="SelectionChangedCommand">
+<ContentPage.BindingContext>
+    <local:CalendarViewModel/>
+</ContentPage.BindingContext>					  
+</calendar:SfCalendar>
+
+{% endhighlight %}
+{% highlight c# tabtitle="MainPage.xaml.cs" hl_lines="3,6,8" %}
+
+public class CalendarViewModel
+{
+    public ICommand SelectionChangedCommand { get; set; }
+    public CalendarViewModel()
+    {
+        SelectionChangedCommand = new Command<CalendarSelectionChangedEventArgs>(SelectionChanged);
+    }
+    private void SelectionChanged(CalendarSelectionChangedEventArgs args)
+    {
+        // To do your requirement here.
+    }
+}
+
+{% endhighlight %}
+{% endtabs %}
+
+### TappedCommand
+
+SfCalendar includes a built-in event called `Tapped` and invokes this event through the [TappedCommand]() whenever the calendar view is tapped, with the `CalendarTappedEventArgs` passed as a parameter.
+
```

Comment:
The `SfCalendar` includes a built-in event called `Tapped` that is triggered whenever the calendar view is tapped. This event can be invoked through the [TappedCommand](), which passes the `CalendarTappedEventArgs` as a parameter.

Reply:
Content updated.